packages/react-components/components/WithPasteAware.tsx

```diff
@@ -20,6 +21,11 @@ interface PasteAwareState {
   clipboardContent: string | null
 }
 
+export function deviceIsIos14OrNewer() {
+  const majorVersionIOS = parseInt(Platform.Version.toString(), 10)
```

Comment:
it it safer to use parseFloat here for iOS 14.1, etc? i guess it does seem like parseInt rounds down (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) but just may be a little confusing

Reply:
My understanding of `parseInt` is that it just dismisses the decimal part, like a cast to int does in other languages, but since it can cause confusion I jut changed it to `parseFloat` :)